install/outline-install.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 community-scripts ORG
+# Author: Gerhard Burger (burgerga)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+read -r -p "Enter the public url for your Outline instance (e.g., https://outline.your-domain.tld)): " URL
```

Comment:
Not a fan of this. Is this url mandatory? does it have to be public? can we use a default? can we use 0.0.0.0?

Reply:
If it is possible to run the app locally, I would do that as default. and people that want to expose publically should set that up themselves.

Reply:
Yes, the url is mandatory and the install will fail otherwise (the url is needed in the `db:migrate` step). https://github.com/outline/outline/blob/5102cfe8eb7f5edd21bd4104e5bfd73f00f0be1d/.env.sample#L30-L33 suggests the URL needs to be public, but I run it fine on my LAN only using a local only access list in Nginx Proxy Manager. Not sure if there are workarounds for this, see https://docs.getoutline.com/s/hosting/doc/ssl-pzk7WO8d1n

Reply:
Having a prompt like this is very error prone and will likely cause a lot of failed installs and issues. I think we must have a default.